src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -301,6 +350,12 @@ export function getFieldRenderer(
   const fieldName = getAggregateAlias(field);
   const fieldType = meta[fieldName];
 
+  for (const alias in SPECIAL_FUNCTIONS) {
```

Comment:
I don't know how to prevent this accidentally kicking in on Discover pages as well. I didn't know if I should add a special renderer unique to the performance page though.

Reply:
Being able to use this rendering on discover seems useful. If the results contain both misery and unique users we can 'upgrade' their UI.